spec/data/v2_for_translation.xml

```diff
@@ -11,8 +11,8 @@
                 <auc:Section ID="Section-1">
                   <auc:Sides>
                     <auc:Side>
-                      <auc:WindowID IDref="Window-1"/>
-                      <auc:DoorID IDref="Door-1"/>
+                      <auc:WindowID IDref="FenestrationSystemType-2"/>
```

Comment:
yikes, the "old" file didn't even reference a valid fenestration :(. Nice catch!